test/webgl/uniforms.spec.js

```diff
@@ -155,14 +178,72 @@ test('WebGL#Uniforms pre verify uniforms', t => {
   t.end();
 });
 
-test('WebGL#Uniforms Program construct/delete', t => {
+test('WebGL#Uniforms Program uniform locations', t => {
   const {gl} = fixture;
 
   const program = new Program(gl, {
     vs: VERTEX_SHADER,
     fs: WEBGL1_FRAGMENT_SHADER
   });
-  t.ok(program instanceof Program, 'Program construction successful');
+
+  for (const uniformName in WEBGL1_GOOD_UNIFORMS) {
+    t.ok(program._uniformSetters[uniformName], `Program found uniform setter ${uniformName}`);
+  }
 
   t.end();
 });
+
+const testSetUniform = gl => t => {
```

Comment:
Nit: I personally find double double arrows as just too compact to easily read. Make one of them a classical function, or at least wrap in `{}`?

Looking at the code, couldn't we just have a function with two parameters?